modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/indices.analyze/10_analyze.yml

```diff
@@ -44,6 +44,32 @@
     - match:  { tokens.0.token: sha }
     - match:  { tokens.1.token: hay }
 
+---
+"Too complex regex pattern":
+    - do:
+        catch: bad_request
+        indices.create:
+          index: test_too_complex_regex_pattern
+          body:
+            settings:
+              index:
+                analysis:
+                  analyzer:
```

Comment:
What is the rectangle shaped char at the end of each line? Can we remove them?

Reply:
GH does not render it well - but this is an actual case where we know `Pattern::compile` throws an `OutOfMemoryError` so I think it is fine to keep it. Might be worth adding a comment here for folks that might read this code in the future? @afoucret

Reply:
I moved this test to a more appropriate place and added a comment to it